os-appium-tests/tests/specs/contact.spec.ts

```diff
@@ -0,0 +1,102 @@
+/*
+After each test scenario (or each suite of tests), the test should have a teardown where all contacts created are deleted
+However, this one doesn't have teardown, because the plugin can't delete contacts and, with appium, we can't access the native contacts and delete them
+*/
```

Comment:
"This can impact other test files, we should be precise with expects" - I think we should add some explanation about the impact of leaving the contacts there.